icu4c/source/i18n/unicode/udisplayoptions.h

```diff
@@ -311,6 +311,19 @@ typedef enum UDisplayOptionsSubstituteHandling {
 
 #endif // U_HIDE_DRAFT_API
 
+#ifndef U_HIDE_DEPRECATED_API
+
+/**
+ * @return the lowercase CLDR keyword string for the grammatical case and `nullptr` if
+ * the grammatical case is undefined.
+ *
+ * @deprecated ICU 72
+ */
+U_CAPI const char *U_EXPORT2
+udispopt_getGrammaticalCaseIdentifierWithNull(UDisplayOptionsGrammaticalCase grammaticalCase);
```

Comment:
OrNull

Reply:
done

Reply:
Adding this function will add a lot of complication for this PR. lets do it in another version then.